package.json

```
@@ -7,16 +7,24 @@
     "type": "git",
     "url": "https://github.com/taymyr/taymyr-antora-ui.git"
   },
+  "scripts": {
+    "preview": "npm-run-all -s build playbook",
+    "playbook": "antora antora-playbook.yml",
+    "build": "webpack"
```

I think that `bundle` is more semantic name and known to those who using Antora Default UI.
```suggestion
    "bundle": "webpack"
```